README.md

```diff
@@ -85,10 +85,10 @@ Scaffold a project using the build tool of your choice. For example, with Gradle
 gradle init --type kotlin-application
 ```
 
-Add the dependency [`sdk-kotlin`](sdk-kotlin):
+Add the dependency [`sdk-api-kotlin`](sdk-api-kotlin):
 
 ```
-implementation("dev.restate.sdk:sdk-kotlin:1.0-SNAPSHOT")
+implementation("dev.restate.sdk:sdk-api-kotlin:1.0-SNAPSHOT")
```

Comment:
Following the above comment: `dev.restate:sdk-kotlin` or `dev.restate:restate-sdk-kotlin` would be simplest, I believe.